aggoracle/e2e_test.go

```diff
@@ -13,19 +13,19 @@ import (
 )
 
 func TestEVM(t *testing.T) {
-	setup := helpers.NewE2EEnvWithEVML2(t, helpers.DefaultEnvironmentConfig())
+	l1Setup, l2Setup := helpers.NewL1SetupWithL2EVM(t, helpers.DefaultEnvironmentConfig(helpers.SovereignChainL2GERContract))
```

Comment:
NewL1SetupWithL2EVM - maybe a better name for this?

Reply:
WDYT `NewSimulatedEnvironmentEVM`? The reason why it was named like this is because we were believing that in the future we might have non-EVM L2 chains and we would need to have separate setups for it

Reply:
Renamed to `NewSimulatedEVMEnvironment` https://github.com/agglayer/aggkit/pull/743/commits/0eb2aa6922cea38182a70b2309b01d36d5db0d91